recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 
 build:
   number: 0
-  skip: True  # [py3k]
+  skip: True  # [osx and py3k]
```

Comment:
> I am not... Google brought up [this](https://github.com/conda-forge/staged-recipes/wiki/conda-smithy-rerender), so should I do this locally and add the result to the PR?

Look at that! I wrote it and forgot about it :flushed: 

That is pretty much it. Let me know if you have any doubt. (I am submitting that wiki to the official docs now, so if you have any feedback please let me know!)